app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -390,6 +396,12 @@ function AcuantCapture(
    */
   function startCaptureOrTriggerUpload(event) {
     if (event.target === inputRef.current) {
+      if (forceNativeCamera) {
+        addPageAction(`IdV: Force native camera. Failed attempts: ${failedCaptureAttempts}`, {
```

Comment:
Since `addPageAction` logs to the event.log, the name passed should be constant and the failed attempts should go in the data hash that will be logged with it.

```suggestion
        addPageAction('IdV: Force native camera.', {
          field: name,
          failed_attempts: $(failedCaptureAttempts),
        });
```